oslabs-beta/DockerLocal
# Refactor app.tsx

## Description:
- Slight refactoring and modularization of AddRepos.tsx file
## Changes I Made:

- [X] Move fetchRepos function to /helpers/fetchRepoHelper.ts 

- [X] Move cookie decryption functionality out of fetchRepos function into modularized cookieClientHelper.ts getUsernameAndToken function. The fetchRepos function now calls getUsernameAndToken function

- [X] Install '@types/crypto-js'

## How to Test:
- Run npm install and then npm start. Click sign in with Github. Toggle isLoggedIn for Dev button. Click Add Repositories button. Search functionality and add repos button should work as previously.
